policy/test/test_connect.py

```diff
@@ -17,9 +17,7 @@ def test_meta_connect(self):
         httpretty.register_uri(httpretty.GET, METADATA_STATUS_URL,
                                body=dumps([]),
                                content_type='application/json')
-        # pylint: disable=protected-access
-        Root._try_meta_connect(0)
-        # pylint: enable=protected-access
+        Root.try_meta_connect(0)
```

Comment:
Just a nitpick here but the default argument is already 0

Reply:
Yes, this is a test and just to make sure we are covering a specific part of the metadata connect logic. So I made sure the attempts was set to zero so we could cover that.